app/models/classroom.rb

```diff
@@ -25,12 +27,30 @@ class Classroom < ApplicationRecord
   has_many :student_classrooms
   has_many :users, through: :student_classrooms
 
-  validates :name, presence: true
+  validates :name, :start_time, :end_time, presence: true
   validate :user_role
+  validate :start_time_after_current_time
+  validate :end_time_after_start_time
+
+  def class_time
+    formatted_time_str = start_time.strftime("%k:%M") + " - " + end_time.strftime("%k:%M")
```

Comment:
you don't need to save it into a variable `formatted_time_str`
that variable won't be accessible anyway